spec/support/input_helpers.rb

```diff
@@ -113,10 +108,26 @@ def expense_resources(**merge)
       },
       expense: {
         e1: expense(merge)
+      },
+      expense_transaction: {
```

Comment:
This could also use some help.  I wish we could use something like `FactoryBot`, but there's no persistence in the lib.  Is there a generic stubbing library we could use that you know of @jozefvaclavik, @mattgmarcus, @saarons?

Reply:
https://github.com/LedgerSync/ledger_sync/issues/32